app/controllers/journals_controller.rb

```diff
@@ -2,6 +2,7 @@ class JournalsController < ApplicationController
   before_action :find_user, only: %i[new create index show destroy update]
   before_action :find_journal, only: %i[show destroy update find_journal]
   before_action :require_login
+  load_and_authorize_resource
```

Comment:
Now that we have load_and_authorize_resource, could you please check the functionality of this line and decide if we need all methods in this file? Referring to the **new** method for example.

Reply:
ok, thanks! I just deleted the new methods where I could. i will investigate more on load_and_authorize_resource to understand what more is unnecessary.

Reply:
Could you please study on this topic and explain how load_and_authorize_resource works?

Reply:
Sure! I investigated some mote time yesterday and replaced the show method aswell.

Reply:
Sure! I investigated some mote time yesterday and replaced the show method aswell.

Reply:
Sure! I investigated some mote time yesterday and replaced the show method aswell.